packages/display/src/Container.js

```diff
@@ -411,12 +411,7 @@ export class Container extends DisplayObject
             this.sortChildren();
         }
 
-        this._boundsID++;
-
-        this.transform.updateTransform(this.parent.transform);
-
-        // TODO: check render flags, how to process stuff here
-        this.worldAlpha = this.alpha * this.parent.worldAlpha;
+        DisplayObject.prototype.updateTransform.call(this);
```

Comment:
Why not just call `this.updateTransform`?

Reply:
We are *in* ``this.updateTransform``. That'd be infinitely recursive.

There used to be a bunch of copy-pasted lines from ``DisplayObject.updateTransform`` here, which weren't really kept in sync thus lead to buggy code when people only updated it in one file. I fixed that here: https://github.com/pixijs/pixi.js/commit/bca4b623a532c79a890d34158ed8228244224d89

Better to not have copy-paste code....

Reply:
Just do `super.updateTransform()`